backend/app/services/kind_impl.py

```
@@ -697,3 +697,76 @@ def _format_resource(self, resource: Kind) -> Dict[str, Any]:
 
         return result
+
+
+class DeviceKindService(KindBaseService):
+    """Service for Device resources (local device management)"""
+
+    def __init__(self):
+        super().__init__("Device")
+
+    def _validate_references(
+        self, db: Session, user_id: int, resource: Dict[str, Any]
```

Ruff (0.14.14)

Unused method argument: `db`

(ARG002)